charts/restate-helm/values.yaml

```diff
@@ -39,18 +39,23 @@ env:
     value: json
   - name: RESTATE_CLUSTER_NAME
     value: helm-single-node
+  - name: RESTATE_ROCKSDB_TOTAL_MEMORY_SIZE
+    # This value should be around 75% of the container memory limit, which defaults to 8Gi below.
+    # If provisioning restate with a different memory limit, make sure to update this value
+    value: 6Gi
```

Comment:
The one alternative I was thinking of is whether we could use K8s downward API to automatically derive the value based on the configured memory limit (specific to the helm chart). But it would also add some form of auto magic (which probably should only happen if the rocksdb total memory hasn't been set explicitly).